stdlib/REPL/test/replcompletions.jl

```diff
@@ -1837,9 +1837,9 @@ let s = "Ref(Issue36437(42))[]."
     c, r, res = test_complete_context(s, @__MODULE__)
     @test res
     for n in ("a", "b", "c")
-        @test n in c
+        @test_broken n in c
     end
-    @test "v" ∉ c
+    @test_broken "v" ∉ c
```

Comment:
This is very unfortunate, but now inference taints `:noub` when seeing `getfield(::Base.RefValue{IsBitsType}, :x)` because of potentially undefined value, causing `REPLInterpreter` to be unable to concrete evaluate it.

Reply:
why was this OK before?

Reply:
We didn't distinguish :noub and :consistent before, and so we've enabled concrete eval for REPL completions even if it may result in undefined behavior. What unfortunate in this PR is that uninitialized value usually doesn't lead to UB but just result in inconsistent results.